server/src/main/java/org/apache/iotdb/db/mpp/execution/QueryExecution.java

```diff
@@ -210,8 +208,7 @@ public TsBlock getBatchResult() {
         return null;
       }
       return resultHandle.receive();
-
-    } catch (ExecutionException | IOException | CancellationException e) {
+    } catch (ExecutionException e) {
```

Comment:
We can remove IOException but we need keep the `CancellationException` here. Because the blocked.get() may throw the CancellationException which is necessary to be caught here.

Reply:
OK, will fix it.